src/VisualStudio/Core/Impl/CodeModel/RootCodeModel.cs

```diff
@@ -16,19 +16,19 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.CodeModel
     [ComDefaultInterface(typeof(EnvDTE.CodeModel))]
     public sealed class RootCodeModel : AbstractCodeModelObject, ICodeElementContainer<AbstractExternalCodeElement>, EnvDTE.CodeModel, EnvDTE80.CodeModel2
     {
-        internal static EnvDTE.CodeModel Create(CodeModelState state, object parent, ProjectId projectId)
+        internal static EnvDTE.CodeModel Create(CodeModelState state, EnvDTE.Project parent, ProjectId projectId)
```

Comment:
@DustinCampbell might recall if there was some bizarre reason we didn't already have this more strongly typed. I only worry since the COM magic here is most strong.